broker/staticconfiguration/api.go

```diff
@@ -10,6 +10,7 @@ type BaseConfig struct {
 	TLSCertFilename                   string        `yaml:"tls_cert_filename"`
 	TLSKeyFilename                    string        `yaml:"tls_key_filename"`
 	AWSCredentialsFilename            string        `yaml:"aws_credentials_filename"`
+	AWSMasterRoleName                 string        `yaml:"aws_master_role_name"`
```

Comment:
I think the name is confusing: it makes me think of the role in the master/InfoSec account.
I'd suggest `AWSListRolesRoleName`.

Reply:
Sure. Let m rename it.